haystack/backends/elasticsearch_backend.py

```diff
@@ -731,6 +757,8 @@ def matching_all_fragment(self):
 
     def add_spatial(self, lat, lon, sfield, distance, filter='bbox'):
         """Adds spatial query parameters to search query"""
+        if six.PY3:
+            long = int
         kwargs = {
             'lat': lat,
             'long': long,
```

Comment:
This change is actually unnecessary – this function has a bug because the variable being passed in is `lon` but the variable being used is the built-in `long` type!